tests/e2e/test.bats

```diff
@@ -39,4 +58,7 @@ ETCD_KEY=/etc/kubernetes/pki/etcd/server.key
 
     result=$(kubectl exec curl -- curl http://${pod_ip}:8787/healthz -o /dev/null -w '%{http_code}\n' -s)
     [[ "${result//$'\r'}" == "200" ]]
+
+    #cleanup
+    run kubectl delete pod curl --force
```

Comment:
Let's also add `--grace-period 0` to ensure the pod isn't stuck in `Terminating` state

Reply:
Good point. done